AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -122,6 +124,14 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
                     self.respond(to: newState)
                 }
         }
+
+        self.outgoingMutationQueueSink = self.outgoingMutationQueue.publisher.sink { [weak self] mutationEvent in
+            guard let self = self else {
+                return
+            }
+            self.log.verbose("MutationEvent received from outgoing mutation queue for topic publisher \(mutationEvent)")
```

Comment:
we should probably remove this log statements since we aren't sending one when we send one in RemoteSyncEngine+IncomingEventReconciliationQueueEvent.

Reply:
sorry, what do u mean?

Reply:
Remove this log statement because in the other case where we are sending a mutation event to the subject, we are do not have a log statement.  (See RemoteSyncEngine+IncomingEventReconciliationQueueEvent.)  Putting a log statement here would make mutation events logging inconsistent.

Reply:
removing